doc/en/user/source/security/urlchecks.rst

```diff
@@ -3,65 +3,172 @@
 URL Checks
 ==========
 
-The URL checks page controls the checks that are performed on user provided URLs that
+The :guilabel:`URL External Access Checks` page controls the checks that are performed on user provided URLs that
```

Comment:
I keep on being surprised about usage of qualifiers like guilabel... why spent time adding them, when the rendering is confusing at best? It renders like a inline code block, the last think I go searching is GUI elements...

![image](https://github.com/geoserver/geoserver/assets/325433/93ab817b-bd7f-4754-a795-cdb62b5f38ec)

Reply:
So you're making that effort only because your vendor needs. Makes sense as long as GeoCat personnel does the effort. But I would remove it from the documentation guidelines then...

Reply:
I wonder if we can update the geoserver theme to be based on rtd theme like everything else these days :)

Reply:
Hum... maybe? We'd need to keep colors and logos, but if that makes full usage of the RST extra qualifiers, why not?